### ISSUE TITLE
Version 1.3.1 has no 32-bit Windows Support

### ISSUE THREAD

**Issue by AMDFreak2006** [processing/processing-sound-archive#49]
Exporting a Processing-program with Sound-library 1.3.1 leads to the following error message:

"The application.windows32 folder will not be created because no 32 version of Sound is available for windows"

A look into the library folder confirms this. No 32-bit Windows folder can be found.

Version 1.2.1 has a 32 bit windows folder. Please fix it :)

**Comment by wirsing** [comment]
There is no 32bit support for Windows at the moment due to larger background issues. This will be noted in the reference soon. When did you encounter that error, when you exported on a Win32 machine?

**Comment by AMDFreak2006** [comment]
No, I exported on windows 7 64 bit but Processing IDE always tries to export for x86 and x64.

**Comment by Kango** [comment]
Hello,

I have processing 3.0 for Win 64 bit now and can't run the sound lib. 

I can't run my sketch. 

It says 

"""""""A lib used by this sketch is not installed properly 

A library relies on native code that's not available.
Or only works properly when the sketch is run as a 32-bit application."""""""""""""""""

---- so it seems, we need to update the lib for Win 64 bit. 

this is my Code: 
  fileSoundForNode = new SoundFile(this, "Conga1.mp3");

I installed the lib anew btw. 

it makes no sense to offer processing for win 64 bit when the core libs for it don't work for Win 64 bit imho

Thanks! Chris
